content/developer/smarty/functions.md

```diff
@@ -327,6 +327,20 @@ Writes the search box to the page.
 
 {{< params "theming/function/searchbox.json" >}}
 
+## Function: `{searchbox_advanced}`
+
+### Usage
+
+```
+{searchbox_advanced placeholder="string"}
```

Comment:
This does not exist. Look at the actual `advancedsearch.php` file, it uses the `t` function with "SearchBoxPlaceHolder" as a key.